packages/framework/src/client.ts

```diff
@@ -261,14 +260,23 @@ export class Client {
   ): ActionStep<T_Outputs, T_Result> {
     return async (stepId, stepResolve, options) => {
       const step = this.getStep(event.workflowId, stepId);
-      const eventClone = cloneData(event);
```

Comment:
Event cloning is no longer needed as the problematic `ajv` validator which mutated the data is now cloning internally.